api-tests/api.test.js

```diff
@@ -89,3 +102,95 @@ test('/events endpoint', async () => {
     ])
   })
 })
+
+test('/events/{eventKey}/info endpoint', async () => {
```

Comment:
Do we want to rely on `2018flor` existing? We could maybe store the first event id from the `/events` test, and then use that in this test.

Reply:
`2018flor` should exist for a very long time. TBA still has events from before 2000 available, afaik they still have every FRC event ever available. I choose `2018flor` since it has specific properties useful for testing (e.g. both a youtube and a twitch webcast) so it will catch bugs that a random event might not. We might actually want to try to test against like 4-5 pre-defined events. I figure if someone is theoretically still using peregrine in 30 years and tba doesn't have `2018flor` anymore, a) they probably won't be hosting api v2 anymore and b) either the person won't need to run the tests or they'll be capable to changing the chosen event. We can make it fall back on the first event if `2018flor` doesn't exist if you want.